integration_tests/tests/admin/nationalOccupancy.cy.ts

```diff
@@ -0,0 +1,56 @@
+import { cruManagementAreaFactory } from '../../../server/testutils/factories'
+import { signIn } from '../signIn'
+import { CRU_AREA_WOMENS } from '../../../server/utils/admin/nationalOccupancyUtils'
+import NationalViewPage from '../../pages/admin/nationaOccupancy/nationalViewPage'
+import ListPage from '../../pages/admin/placementApplications/listPage'
+
+context('National occupancy view', () => {
+  const cruManagementAreas = [
+    ...cruManagementAreaFactory.buildList(5),
+    cruManagementAreaFactory.build({ id: CRU_AREA_WOMENS }),
+  ]
+
+  const filterSettings = {
+    postcodeArea: 'SW13A',
+    arrivalDate: '12/9/2024',
+    apArea: cruManagementAreas[3].id,
+    apType: 'pipe',
+    apCharacteristics: ['isCatered'],
+    roomCharacteristics: ['isWheelchairDesignated'],
+  }
+
+  beforeEach(() => {
+    cy.task('reset')
+
+    // Given I am signed in as a CRU member
+    signIn('cru_member')
+    cy.task('stubCruManagementAreaReferenceData', { cruManagementAreas })
+    cy.task('stubPlacementRequestsDashboard', { placementRequests: [], status: 'notMatched' })
+  })
+
+  it('allows me to set up filters for the national view', () => {
+    cy.log('Given that I am on the CRU dashboard page')
```

Comment:
What do you think about this?
I'm fed up with trying to fix broken integration tests and having no idea where the test is breaking - so I though if I change the comments to logs ...